tests/integration/nm/wired_test.py

```diff
@@ -35,15 +35,12 @@
 MTU0 = 1200
 
 
-@pytest.mark.xfail(
-    condition=StrictVersion(nm.nmclient.nm_version()) < StrictVersion("1.18"),
-    reason="https://bugzilla.redhat.com/1702657",
-    strict=True,
-)
+@pytest.mark.xfail(reason="Ref: https://bugzilla.redhat.com/1797986")
```

Comment:
The `Ref: ` prefix can be removed here and in the other lines

Reply:
Done, thanks!